summa/keywords.py

```diff
@@ -216,9 +216,9 @@ def keywords(text, ratio=0.2, words=None, language="english", split=False, score
     return _format_results(keywords, combined_keywords, split, scores)
 
 
-def get_graph(text, language="english"):
+def get_graph(text, language="english", deaccent=False):
     tokens = _clean_text_by_word(text, language)
```

Comment:
The way you get the tokens should be coherent with the way you split the text. I think the right way should be:
```
tokens = _clean_text_by_word(text, language, deacc=deaccent)
split_text = list(_tokenize_by_word(text, deacc=deaccent))
```

Reply:
Thanks for the catch, updated now.